Lesson4/Building/BuildingInfo.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace Lesson4.Building
+{
+    public class BuildingInfo
+    {
+        private static int numberBuilding = 1;
+       
+        public double Height { get; private set; }
+       
+        public int Floor { get; }
+        
+        //Кол-во квартир на этаже
+        public int Flats { get; }
+        
+        //Кол-во подъездов в доме
+        public int Entrance { get; }
```

Comment:
А почему оставшиеся свойства тогда без комментариев?
```suggestion
        /// <summary>Кол-во квартир на этаже</summary>
        public int Flats { get; }
        
        /// <summary>Кол-во подъездов в доме</summary>
        public int Entrance { get; }
```